src/index.js

```diff
@@ -339,6 +339,25 @@ class LawReg {
     }
   }
 
+  /**
+   * Extract the DIDs being referred, if the input is an OR Function or a string with IS-constructions
+   *
+   * @param {ParsedExpression | string} functionRef - Possible OR Function or string with IS-constructions
+   * @returns {string[]} - The DIDs being referred if it is an IS-construction
+   */
+  static extractDids (functionRef) {
```

Comment:
Given how the did-check is currently done in the checkFactLink function I get why you went this way, but I think it's a little funky to have the explicit check for OR in this place. Do you feel it's worth investigating if we can refactor the existing code such that we can resolve the OR using the function we already have and then delegate to existing logic to just check a single did?

Reply:
I agree.